perun-core/src/main/resources/oracleChangelog.txt

```diff
@@ -6,6 +6,11 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.1.52
+create or replace type TARRAYOFCHARACTERS is table of  nvarchar2(100);
```

Comment:
Maybe we don't need to restrict ourselves with limits to character length or number size ?, also unnecessary whitespace.